vscode-pets-integration/experiment1/scripts/sprite-generation/generate_forkmonkey.py

```
@@ -0,0 +1,240 @@
+            if part.inline_data:
+                image_bytes = part.inline_data.data
+                image = Image.open(io.BytesIO(image_bytes))
+                
+                # Save frame
+                frame_path = os.path.join(OUTPUT_DIR, f"{animation_name}_frame_{frame_number:02d}.png")
+                image.save(frame_path)
+                print(f"  ✅ Saved: {frame_path}")
+                
+                return image
+        
+        raise Exception("No image in response")
```

Ruff (0.14.10)

Abstract `raise` to an inner function

(TRY301)